packages/lesswrong/components/recommendations/RecommendationsAndCurated.tsx

```diff
@@ -143,86 +182,133 @@ const RecommendationsAndCurated = ({
 
     const bookmarksLimit = (settings.hideFrontpage && settings.hideContinueReading) ? 6 : 3
 
-    return <SingleColumnSection className={classes.section}>
-      <AnalyticsContext pageSectionContext="recommendations">
-        <SectionTitle title={<LWTooltip title={recommendationsTooltip} placement="left">
-          <Link to={"/recommendations"}>
-            {isEAForum ? 'Classic posts' : 'Recommendations'}
-          </Link>
-        </LWTooltip>}>
-          {!isEAForum && currentUser &&
+    const titleText = isEAForum ? "Classic posts" : "Recommendations"
+    const titleNode = (
+      <div className={classes.title}>
+        <SectionTitle
+          title={
+            <>
+              {isEAForum ? (
+                <>{ titleText }</>
+              ) : (
+                <LWTooltip title={recommendationsTooltip} placement="left">
+                  <Link to={"/recommendations"}>{titleText}</Link>
+                </LWTooltip>
+              )}
+              {isEAForum && (
+                <ForumIcon
+                  icon={sectionExpanded ? "ThickChevronDown" : "ThickChevronRight"}
+                  onClick={toggleSectionVisibility}
+                  className={classes.expandIcon}
+                />
+              )}
+            </>
+          }
+        >
+          {!isEAForum && currentUser && (
             <LWTooltip title="Customize your recommendations">
-              <SettingsButton showIcon={false} onClick={toggleSettings} label="Customize"/>
+              <SettingsButton showIcon={false} onClick={toggleSettings} label="Customize" />
             </LWTooltip>
-          }
+          )}
+          {isEAForum && sectionExpanded && (
+            <Link to="/recommendations" className={classes.readMoreLink}>
+              View more
+            </Link>
```

Comment:
Sorry this section ended up as a bit of a mess of forum gating, this could probably be refactored to be simpler. Also in the community section there is a separate element for mobile with a readMoreLinkMobile class, but I don't think it's working properly so I haven't duplicated that behaviour here. I've checked and it looks ok on mobile

Reply:
I'm kinda ok with it, given that we will revisit soon